### PR TITLE
[TASK] Add note about breaking flex form section change

### PR DESCRIPTION
**Pull request by garvinhicking**
type="select" is no longer allowed.
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/836

### CODE REVIEW

**Review by linawolf — DISMISSED** [review]

File: Documentation/ApiOverview/DataFormats/T3datastructure/Elements/Index.rst [hunk 1]
```diff
@@ -182,9 +182,23 @@ must be strings or integers.)
          Defines for an object of the type <array> that it must contain other
          "array" type objects in each item of <el>. The meaning of this is application specific. For
          FlexForms it will allow the user to select between possible arrays of
-         objects to create in the form. For TemplaVoila it will select a
-         "container" element for another set of elements inside. This is quite
-         fuzzy unless you understand the contexts.
+         objects to create in the form. This is similar to the concept of
+         :ref:`IRRE / inline TCA definitions <t3tca:columns-inline>`.
+         For TemplaVoila it will select a "container" element for another set
```
Comment (brotkrueml) [inline Documentation/ApiOverview/DataFormats/T3datastructure/Elements/Index.rst:187]: Do we really need to mention Templavoila?

Comment (garvinhicking) [reply Documentation/ApiOverview/DataFormats/T3datastructure/Elements/Index.rst:187]: People stil seem to use it and I can't comment on how useful it is here in the docs. Not sure if I can extrapolate my own usage of TV here - I'm ok if you recommend to remove it!

Comment (linawolf) [reply Documentation/ApiOverview/DataFormats/T3datastructure/Elements/Index.rst:187]: Yes, please remove it is not useful here in the docs anymore

Comment (garvinhicking) [reply Documentation/ApiOverview/DataFormats/T3datastructure/Elements/Index.rst]: Changed.

**Pull request merged**